DEVDOC.md

```diff
@@ -116,6 +116,42 @@ lookups.
 
 Any help to fix this is very welcome!
 
+#### Integration with VSCode and Delve
+
+If you would like to use the delve debugger for Go with VSCode to debug `gapis`, You can follow these steps
+
+1. Complete the fusing steps.
+
+2. Create a launch.json under the project directory `Ctrl + Shift + P` and `Debug: Open launch.json`
+
+3. Paste this as one of the configuration. This will ensure that you have a setup for attaching to Delve when you launch the project.
+```
+{
```

Comment:
Indentation of this JSON is a bit weird (outmost brackets are not idented the same), maybe it would be better to put an abbreviated version of the surrounding JSON, e.g.:
```
{
    ...
    "configurations": [
        ...
        {
            "name": "Attach to Delve",
            "type": "go",
            "request": "attach",
            "mode": "remote",
            "apiVersion": 2,
            "remotePath": "gapis/",
            "cwd": "${workspaceFolder}/src/github.com/google/gapid/gapis",
            "dlvLoadConfig": {
                "followPointers": true,
                "maxVariableRecurse": 1,
                "maxStringLen": 120,
                "maxArrayValues": 120,
                "maxStructFields": -1
            },
            "host": "127.0.0.1",
            "port": 1234,
        }
    ]
}
```